docs/data/external-data/ga-srtm-1-second-dem/_data.yaml

```
@@ -3,7 +3,7 @@
 
 # Overview
 
-title: SRTM 1 second DEM
+title: GA SRTM 1 second DEM
 long_title: SRTM 1 second DEM version 1.0
```

Maybe the following to spell out the acronymn and add "Geoscience Australia"?

```suggestion
long_title: Geoscience Australia Shuttle Radar Topography Mission 1 second DEM version 1.0
```